.circleci/config.yml

```diff
@@ -321,11 +321,24 @@ workflows:
                   name: << matrix.theme >>-<< matrix.color >>-<< matrix.scale >>-<< matrix.dir >>
                   matrix:
                       parameters:
-                          theme: [classic, express]
+                          theme: [spectrum, express]
```

Comment:
Just an FYI. This changes the order of the Circle CI jobs in the UI and can make visually checking "did everything work, but the VRTs are unhappy" hard to differentiate from "things are broken AND the VRTs are unhappy"...

<img width="403" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/6d123854-564a-41d8-bb5c-230079862bfa">

vs

<img width="310" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/74789522-495b-41f3-8b81-a18bec3bf0cf">